runtime/queries/typst/highlights.scm

```diff
@@ -0,0 +1,79 @@
+(call
+  item: (ident) @function)
+(call
+  item: (field field: (ident) @function.method))
+(tagged field: (ident) @tag)
+(field field: (ident) @tag)
+(comment) @comment
+
+; CONTROL
+(let "let" @keyword.storage.type)
+(branch ["if" "else"] @keyword.control.conditional)
+(while "while" @keyword.control.repeat)
+(for ["for" "in"] @keyword.control.repeat)
+(import "import" @keyword.control.import)
+(as "as" @keyword.operator)
+(include "include" @keyword.control.import)
+(show "show" @keyword.control)
+(set "set" @keyword.control)
+(return "return" @keyword.control)
+(flow ["break" "continue"] @keyword.control)
+
+; OPERATOR
+(in ["in" "not"] @keyword.operator)
+(and "and" @keyword.operator)
+(or "or" @keyword.operator)
+(not "not" @keyword.operator)
+(sign ["+" "-"] @operator)
+(add "+" @operator)
+(sub "-" @operator)
+(mul "*" @operator)
+(div "/" @operator)
+(cmp ["==" "<=" ">=" "!=" "<" ">"] @operator)
+(fraction "/" @operator)
+(fac "!" @operator)
+(attach ["^" "_"] @operator)
+(wildcard) @operator
+
+; VALUE
+(raw_blck "```" @operator) @markup.raw.block
+(raw_span "`" @operator) @markup.raw.block
+(raw_blck lang: (ident) @tag)
+(label) @tag
+(ref) @tag
+(number) @constant.numeric
+(string) @string
+(content ["[" "]"] @operator)
+(bool) @constant.builtin.boolean
+(builtin) @constant.builtin
+(none) @constant.builtin
+(auto) @constant.builtin
+(ident) @variable
+(call
+  item: (builtin) @function.builtin)
+
+; MARKUP
+(item "item" @operator)
```

Comment:
@AlexanderBrevig  56,57 reports an error with latest tree-sitter-typst.
```
(item "item" @operator) 
(term ["item" ":"] @operator)
```
here is the error message:
```
". Are your grammars out of sync? Try running 'hx --grammar fetch' and 'hx --grammar build'. This query could not be parsed: QueryError { row: 56, column: 8, offset: 1424, message: "
```